src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewTransferNoteViewModelMapTransfer.cs

```diff
@@ -20,6 +20,8 @@ public class ViewTransferNoteViewModelMapTransfer
 
         public DateTime SystemDateTime { get; set; }
 
+        public bool IncludeHistory { get; internal set; }
```

Comment:
Can be removed a transfer note wont have the history its the original evidence note that has the history. George to update the wireframe.

Reply:
done